packages/env/lib/config/test/config.js

```diff
@@ -251,17 +264,17 @@ describe( 'readConfig', () => {
 				pluginSources: [
 					{
 						type: 'local',
-						path: expect.stringMatching( /^\/.*relative$/ ),
+						path: expect.stringMatching( /^(\/||\\).*relative$/ ),
```

Comment:
Would it be easier to match based on `path.sep` for these tests?

Reply:
if we use variables( `path.sep` ) in a regular expression, we need to generate a `regExp`.
I used this method because I think `newRregExp` makes the code verbose.

Also, I could not figure out how to embed a non-escaping backslash in the regexp 😅

Reply:
Thanks, that makes sense!